developers.libra.org/scripts/build_docs.sh

```diff
@@ -26,8 +28,19 @@ function install_rustup {
   fi
 }
 
+function install_python {
```

Comment:
This function seems in properly named since it's not actually installing python but just checking that it is installed

Reply:
Good point, can change.